### ISSUE TITLE
CLI commands cannot find the text files if run outside of the repo directory

### ISSUE THREAD

**Issue by fdmarcin** [fdmarcin/benerator-cumberpy#7]
Regardless of installation method (locally or from PyPI), the `benerate` and `hello-ben` commands only work if you're in the `benerator-cumberpy` directory.

If you run either from anywhere else, you'll get an error:

```py
Traceback (most recent call last):
  File "/home/marcin/.local/bin/hello-ben", line 8, in <module>
    sys.exit(hello())
  File "/home/marcin/.local/lib/python3.8/site-packages/benerator_cumberpy.py", line 25, in hello
    print("Hello there, my name is " + benerate_name() + ".")
  File "/home/marcin/.local/lib/python3.8/site-packages/benerator_cumberpy.py", line 9, in benerate_name
    with open('data/first-names.txt') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'data/first-names.txt'
```

It looks like the way it's written in [benerator_cumberpy.py](https://github.com/fdmarcin/benerator-cumberpy/blob/master/benerator_cumberpy.py), the script can't access the text files when executed from somewhere else.

**Comment by fdmarcin** [comment]
I think the issue is two-fold:

- The `data` folder was not included in the installation - fixed by https://github.com/fdmarcin/benerator-cumberpy/commit/dc7bff4e1deecefd2bd64ebe52892733327ca461
- The way I refer to the files in the code prevents updating the paths when installed.

**Comment by fdmarcin** [comment]
Fixed in 0.2.1: https://github.com/fdmarcin/benerator-cumberpy/commit/dfcddae7cca2ad5e686da39adae35862413d8795 :tada: :tada: :tada:
